pkg/api/routes.go

```diff
@@ -904,6 +904,12 @@ func (rh *RouteHandler) PatchBlobUpload(w http.ResponseWriter, r *http.Request)
 
 	if err != nil {
 		switch err {
+		case io.ErrUnexpectedEOF:
```

Comment:
Is it possible to be interrupted also in UpdateBlobUpload function? (see line 1015)